.gitignore

```diff
@@ -5,3 +5,6 @@ npm-debug.log
 .DS_Store
 docs/
 !scripts/docs/
+
+# Ignore Argos CI screenshots.
+functional-tests/screenshots/
```

Comment:
Can it be only dealt with by travis, i.e. what's the use of git keeping this folder in dev env?